test/WebSites/RazorPagesWebSite/StartupWithBasePath.cs

```diff
@@ -18,6 +18,7 @@ public void ConfigureServices(IServiceCollection services)
                     options.RootDirectory = "/Pages";
                     options.AuthorizePage("/Conventions/Auth");
                     options.AuthorizeFolder("/Conventions/AuthFolder");
+                    options.SetPageName("/Routes/NameConfiguredAtStartup", "StartupName");
```

Comment:
See my question in the work item about the need for the slash discrepancy here versus the name.